src/loaders/lottie/tvgLottieModel.cpp

```diff
@@ -141,7 +141,9 @@ void LottieImage::prepare()
 void LottieTrimpath::segment(float frameNo, float& start, float& end, LottieExpressions* exps)
 {
     start = this->start(frameNo, exps) * 0.01f;
+    start = start < 0.0f || start > 1.0f ? 0.0f : start;
```

Comment:
Why the exceed values are exported from the get go....

please do not assign data unnecessarily.
```if (start < 0.0f || start > 1.0f ? 0.0f) start = 0.0f;```